discv5/crawler.go

```diff
@@ -25,6 +25,8 @@ import (
 	"github.com/dennis-tra/nebula-crawler/discvx"
 )
 
+const MAX_CRAWL_RETRY_AFTER_TIMEOUT = 2 // magic
```

Comment:
Nit: Usually Go's constants follow a different style than in many other languages: https://google.github.io/styleguide/go/decisions.html#constant-names

Reply:
Addressed